docs/Installation.md

```diff
@@ -0,0 +1,71 @@
+## Installation
+We suggest to install or use the package in the [Python virtual environment](https://docs.python.org/3/tutorial/venv.html).
+
+If you want to optimize a model from PyTorch, install PyTorch by following [PyTorch installation guide](https://pytorch.org/get-started/locally/#start-locally). 
+If you want to optimize a model from TensorFlow, install TensorFlow by following [TensorFlow installation guide](https://www.tensorflow.org/install/).
+
+#### As a package built from a checked-out repository:
+
+Install the package and its dependencies by running the following command in the repository root directory:
+```
+pip install .
+```
+
+Note that if you install NNCF in this manner, the backend frameworks supported by NNCF will not be explicitly installed. NNCF will try to work with whatever backend versions you have installed in your Python environment.
+
+If you want to install both NNCF and the supported PyTorch version in one line, you can do this by running:
+```
+pip install .[torch]
```

Comment:
How about do it shorter. It's more for nncf develop team, other users install backend via own requirements list, no need to describe it too detailed.

Something like:

```
If you want to install NNCF and one of supported backend version in one line, you can do this by running:
`pip install .[<BACKEND>]`
List of  supported backends: `torch`, `tf`, `onnx` and `openvino`.

For development install extra packages by `pip install .[dev,tests]`.
```

Reply:
Removed this part altogether, that's even shorter.

Reply:
But still 8/10 command lines are only for developer of nncf.